provider/source/data/debug/units/displaynames@1/long-foot/ar.json

```diff
@@ -1,13 +1,14 @@
 {
   "patterns": {
     "strings": {
-      "specials": [
-        [
-          "One",
-          "\u0000قدم"
-        ]
+      "one": [
+        0,
```

Comment:
No, but I don't have a super-clean way to remove it. It is the serialization of the tuple `(FourBitMetadata, &str)`. In the string case specifically, I can special-case and detect the string, but that doesn't work generally for a `VarULE` because the `VarULE` could itself serialize to `seq`, for example.

I could get most of the way there by changing it to a dictionary and sniffing for a dictionary with a `metadata` field, but it is still possible for a VarULE to serialize to a map having a field with that name.

Reply:
I see. Adding the `FourBitMetadata` seems like a very premature optimisation, as I don't see it being used anywhere. It also restricts us from adding more explicit plural variants, so I don't really want to commit to it.